store/map/index.js

```diff
@@ -190,6 +190,17 @@ export const getters = {
       .filter(identity)
   },
 
+  activeTimestamp(state, { activeSpatialData }) {
+    if (activeSpatialData.length) {
+      const str = activeSpatialData[0]
+      const timestamp = str.split(/time=([^&]+)/)[1]
```

Comment:
try to make this a bit more resilient, if the `split` does not result in at least an array with two items, this will break because you access `[1]`. you could use `head` here